src/stock-adjustment-mobile/edit-product-page/edit-product-page.jsx

```diff
@@ -365,6 +361,14 @@ const EditProductPage = ({ offlineService, adjustmentType, setToastList, setAdju
                                                     label="Quantity"
                                                     containerClass='field-full-width required'
                                                 />
+                                                <BaseField
+                                                    renderInput={inputProps => (<DateInput {...inputProps}/>)}    
+                                                    maxDate={maxDateToday}
+                                                    name={`${name}.occurredDate`}
+                                                    label="Date"
+                                                    containerClass='field-full-width required'
```

Comment:
Not `field-full-width-last`?

Reply:
No, it is old approach. Could be without "last"